src/boc/CellParsing.js

```diff
@@ -0,0 +1,285 @@
+const {BN} = require("../utils");
+const {BitString} = require("./BitString");
+
+/**
+ * Loads Bit
+ * 
+ * @throws {Error} Error object with description
+ * @param {Cell} slice object to parse
+ * @returns {number}
+ */
+function loadBit(slice) {
+    return slice.bits.readBit();
+}
+
+/**
+ * Loads Bits
+ * 
+ * @throws {Error} Error object with description
+ * @param {Cell} slice object to parse
+ * @returns {BitString}
+ */
+function loadBits(slice, n) {
+    return slice.bits.readBits(n);
+}
+
+/**
+ * Loads Uint
+ * 
+ * @throws {Error} Error object with description
+ * @param {Cell} slice object to parse
+ * @returns {BN}
+ */
+function loadUint(slice, n) {
+    if (slice.bits.readCursor + n > slice.bits.writeCursor) {
+        // read operation would through anyway, but here we get more
+        // detailed error
+        throw Error("cannot load uint: no enough bits");
+    }
+    return slice.bits.readUint(n);
+}
+
+/**
+ * Loads signed int
+ * 
+ * @throws {Error} Error object with description
+ * @param {Cell} slice object to parse
+ * @returns {BN}
+ */
+function loadInt(slice, n) {
+    if (slice.bits.readCursor + n > slice.bits.writeCursor) {
+        // read operation would through anyway, but here we get more
+        // detailed error
+        throw Error("cannot load int: no enough bits");
+    }
+    return slice.bits.readInt(n);
+}
+
+/**
+ * Loads unsigned integer less or equal n by reading minimal number of bits encoding n
+ * 
+ * #<= p
+ * 
+ * @param {Cell} slice object to parse
+ * @param {number} n parameter
+ * @returns {BN}
+ */
+function loadUintLEQ(slice, n) {
+  var result = loadUint(slice, ((new BN(n)).add(new BN(1))).toString(2).length);
```

Comment:
The `.add(new BN(1))` can be replaced with `.addn(1)` for simplicity.

Also, its better to use `const` instead of `var`.